backend/molgenis-emx2-semantics/src/main/java/org/molgenis/emx2/beaconv2/common/EntryType.java

```diff
@@ -0,0 +1,64 @@
+package org.molgenis.emx2.beaconv2.common;
+
+import com.fasterxml.jackson.annotation.JsonAutoDetect;
+import org.molgenis.emx2.beaconv2.endpoints.configuration.DefaultSchema;
+import org.molgenis.emx2.beaconv2.endpoints.configuration.OntologyTermForThisType;
+
+@JsonAutoDetect(fieldVisibility = JsonAutoDetect.Visibility.ANY)
+public class EntryType {
+
+  private String id;
+  private String name;
+  private String partOfSpecification;
+  private String description;
+  private DefaultSchema defaultSchema;
+  private OntologyTermForThisType ontologyTermForThisType;
+
+  public EntryType(
+      String entryTypeSingular,
+      String entryTypePlural,
+      String name,
+      String description,
+      String ontologyId,
+      String ontologyLabel) {
+    this.id = entryTypeSingular;
+    this.name = name;
+    this.partOfSpecification = "Beacon v2.0.0-draft.4";
+    this.description = description;
+    this.defaultSchema = new DefaultSchema(entryTypeSingular, entryTypePlural);
+    this.ontologyTermForThisType = new OntologyTermForThisType(ontologyId, ontologyLabel);
+  }
+
+  // todo make richer? e.g.
+  //  Map<String, String> aCollectionOf =
+  //      new HashMap<String, String>() {
```

Comment:
There are indeed a few places with comments that must be removed in a final version. Suggest to keep them for now in this draft version because they contain pointers and reminders for implementing the framework.